lighthouse-plugin-publisher-ads/messages/en-US.json

```diff
@@ -42,20 +42,20 @@
       "title": "Latency of first ad request",
       "failureTitle": "Reduce latency of first ad request",
       "description": "This metric measures the elapsed time from the start of page load until the first ad request is made. Delayed ad requests will decrease impressions and viewability, and have a negative impact on ad revenue. [Learn more](https://developers.google.com/publisher-ads-audits/reference/audits/metrics).",
-      "displayValue": "%s s"
+      "displayValue": "{adReqTime, number, seconds} s"
```

Comment:
Nit: remove space to be consistent with line 20 above.

Reply:
I'll add the space to line 20 for consistency with lighthouse core